### ISSUE TITLE
5.6.1 (5/5) divide and conquer ?

### ISSUE THREAD

**Issue by blieberger** [pat-rogers/Ada-202x-WG9-Informal-Review#19]
Divide and conquer algorithms are usually recursive. This is not the case here.
Delete comment?

Johann

**Comment by sttaft** [comment]
I would agree they are usually recursive, but not always.  So I think it is fine as is.

**Comment by stephen-baird-751** [comment]
I agree with Tuck.

**Comment by ARG-Editor** [comment]
I agree as well, so no action is planned on this one.
